### PR TITLE
feat: click on bookmark name to view it

### PR DESCRIPTION
**Pull request by QuCMGisaia**
- Fix gisaia/ARLAS-WUI#875

Allow for the bookmark name to be clicked. It is highlighted on hover.

Side changes:
- css -> scss
- bookmark -> signet in french translation
- add description for bookmark combination

image

### CODE REVIEW

**Comment by MohamedHamouGisaia** [comment]
@QuCMGisaia , I suggest too to make the name selectable and opens the bookmark
- the hover of the name should underline the name and the cursor should be pointer

does it make sense  to you ?

**Review by MohamedHamouGisaia — APPROVED** [review]

**Review by MAudelGisaia — APPROVED** [review]

**Comment by QuCMGisaia** [comment]
> @QuCMGisaia , I suggest too to make the name selectable and opens the bookmark
> 
> * the hover of the name should underline the name and the cursor should be pointer
> 
> does it make sense to you ?

I think it does make sense yes! In that case do you think I should keep the eye button visible or should I hide it as well ? I prefer keeping it personally, but I am open to suggestions

**Comment by MohamedHamouGisaia** [comment]
for that i think we should verify with matthieu as we already proposed the bookmark with only the menu to the client

File: projects/arlas-toolkit/assets/i18n/fr.json [hunk 1]
```diff
@@ -38,13 +38,14 @@
 	"Remove": "Supprimer",
 	"Date": "Date",
 	"Name": "Nom",
-	"Add a dataset": "Ajouter un bookmark",
```
Comment (MohamedHamouGisaia) [inline projects/arlas-toolkit/assets/i18n/fr.json:41]: I don't know if we should use the french word for bookmarks :/

**Review by MohamedHamouGisaia — APPROVED** [review]

**Review by MAudelGisaia — APPROVED** [review]

File: projects/arlas-toolkit/assets/i18n/fr.json (hunk 1 above)
Comment (QuCMGisaia) [reply projects/arlas-toolkit/assets/i18n/fr.json:41]: If we think that bookmark is a better choice, I can revert those changes !

**Review by MohamedHamouGisaia — APPROVED** [review]

**Review by MAudelGisaia — APPROVED** [review]

**Pull request merged**